ckanext/harvester_dashboard/helpers.py

```diff
@@ -0,0 +1,15 @@
+from ckan.lib.helpers import lang
+import ckan.plugins.toolkit as tk
+
+def harvester_dashboard_organization_title(value):
+    """display the organization title"""
+    try:
+        localized_value = value.get(lang())
+        if localized_value:
+            return localized_value
+        locales = tk.config.get('ckan.locales_offered')
+        if locales:
+            for locale in locales:
+                return value.get(locale)
```

Comment:
if there isn't a value for the first locale in the locales list, this will return `None`, even if there would be a value for one of the other locales. i think you should only return `value.get(locale)` if `value.get(locale) is not None`.